tests/testdata/Dockerfile-postgis

```diff
@@ -0,0 +1,7 @@
+FROM kartoza/postgis-11.0-2.5
+
+ADD auth_system/certs_keys/localhost_ssl_cert.pem /etc/ssl/certs/localhost_ssl_cert.pem
+ADD auth_system/certs_keys/localhost_ssl_key.pem /etc/ssl/private/localhost_ssl_key.pem
+ADD auth_system/certs_keys/chains_subissuer-issuer-root_issuer2-root2.pem /etc/ssl/certs/chains_subissuer-issuer-root_issuer2-root2.pem
+
+ADD temp/setup-ssl.sh /
```

Comment:
Note to self: once this works, ask kartoza for a new release with https://github.com/kartoza/docker-postgis/pull/129

Reply:
CC @timlinux